docs/user-guide/alert-manager.rst

```diff
@@ -34,8 +34,8 @@ Below is an example AlertManager configuration. Depending on your setup, the exa
         route:
           routes:
           - receiver: 'robusta'
-            match_re:
-              severity: 'info|warn|error|critical'
+          matchers:
```

Comment:
The indentation here is not right.

Reply:
Thanks for pointing it out @pavangudiwada. I have fixed and pushed the changes.